frontend/src/components/forgot-password/styles.module.scss

```diff
@@ -0,0 +1,94 @@
+.bgImage {
+  position: relative;
+  z-index: 1;
+  background-position: center center;
+  background-size: cover;
+}
+
+.bgImage::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: -1;
+  width: 100%;
+  height: 100%;
+  background-color: var(--blue300);
+  opacity: 0.5;
+}
+
+.wrapper {
+  height: 100vh;
+}
+
+.wrapperInner {
+  display: flex;
+  height: 100%;
+}
+
+.content {
+  width: 538px;
+  max-width: 100%;
+  margin: auto;
+  padding: 60px 90px;
+  background: var(--white);
+  border-radius: 10px;
+}
```

Comment:
Could you create a generic wrapper for such content? Because I see that these styles are duplicated. Please get in touch w/ @setar92 as he has the same styles for his email confirmation windows